tasks/main.yml

```diff
@@ -13,6 +14,16 @@
       paths:
         - "{{ role_path }}/vars"
 
+- name: Get servies list using platform specific services manager
+  include_tasks: "{{ item }}"
+  with_first_found:
+    - "services-el{{ ansible_facts['distribution_major_version'] }}.yml"
```

Comment:
```yaml
- name: Get services list using platform specific services manager
  include_tasks: "{{ lookup('first_found', __timesync_ff_params) }}"
  vars:
    __timesync_ff_params:
      files:
        - "{{ ansible_facts['distribution'] }}_{{ ansible_facts['distribution_major_version'] }}.yml"
        - "default.yml"
      paths:
        - "{{ role_path }}/tasks/setup"
```
This is based on https://github.com/oasis-roles/meta_standards#platform-specific-tasks
Note that you have to specify both the distribution and version - otherwise on e.g. Fedora `"services-el{{ ansible_facts['distribution_major_version'] }}.yml"` will evaluate to `services-el32.yml` which is incorrect.

With the above, you'll need to create a new directory `tasks/setup/` which will contain 3 files - `CentOS-6.yml`, `RedHat-6.yml`, and `default.yml` - the first two files will use essentially what you have in services-el6.yml , and `default.yml` will contain essentially what you have in services-el7.yml.

Also note that the register variable is global, so must be prefixed by the role name - so `timesync_services`